api/api_full.go

```diff
@@ -837,10 +837,13 @@ type SyncState struct {
 
 type SyncStateStage int
 
+// FIXME: Review names. Tipset instead of headers. Describe the action
+//  not the object (fetch and validate instead of headers and messages).
 const (
 	StageIdle = SyncStateStage(iota)
+	// Fetch tipsets.
 	StageHeaders
-	StagePersistHeaders
```

Comment:
We should leave it here, for now, removing it will re-number the enum resulting in incompatibility.
We should open an issue for API breaking changes we want to do, mention this renumbering and link to the issue from here.